tools/tests/apps/failover-test/server/failover-test.js

```diff
@@ -69,7 +69,7 @@ C.find().observeChanges({
       Meteor.clearTimeout(nextStepTimeout);
       nextStepTimeout = null;
     }
-    if (!fields.step && _.has(steps, fields.step)) {
+    if (!fields.step && steps.hasOwnProperty(fields.step)) {
```

Comment:
Generally we prefer `Object.prototype.hasOwnProperty.call(steps, fields.step)`, but since this is a test and it passes, I think this is fine. The rationale is that `hasOwnProperty` could be a field in the `steps` object, which would mean `steps.hasOwnProperty !== Object.prototype.hasOwnProperty`.

Reply:
Ah, I was wondering why I was seeing it used like that elsewhere. I will update my other PRs as well. Thanks!